src/registrar/migrations/0090_wafflesample_waffleswitch_waffleflag.py

```diff
@@ -0,0 +1,214 @@
+# Generated by Django 4.2.10 on 2024-05-01 15:10
+
+from django.conf import settings
+from django.db import migrations, models
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("auth", "0012_alter_user_first_name_max_length"),
```

Comment:
**comment**: interesting it added this one. I assume it's just to show it needs django auth intact, but since that's already been a dependency before curious it gets added again